bundles/binding/org.openhab.binding.tcp/src/main/java/org/openhab/binding/tcp/protocol/internal/ProtocolGenericBindingProvider.java

```diff
@@ -59,7 +61,14 @@
     private static final Pattern ACTION_CONFIG_PATTERN = Pattern
             .compile("(<|>)\\[(.*?):(.*?):(.*?):(?:'(.*)'|(.*))\\]");
     private static final Pattern STATUS_CONFIG_PATTERN = Pattern.compile("(<|>)\\[(.*?):(.*?):(?:'(.*)'|(.*))\\]");
+    
+    /**
+     * Artificial command to identify that state changes should be taken into account
+     */
+    private static final Command CHANGED_COMMAND_KEY = StringType.valueOf("CHANGED");
```

Comment:
I don't think "CHANGED" makes sense in the context of receiving commands.  If you agree, please remove definition/handling of it.